### DISCUSSION TITLE
Clone project without project main folder.

### DISCUSSION THREAD

**Discussion by hakanoktay** [gitpython-developers/GitPython#1168] [Q&A] [answered]
-- Hi,
-- how can i clone only files and directories? When i clone, it downloading in a folder named by project name.
/path/to/xxxxx

```
# Example usage
import git
git.Git(WEBDIR + domain.name + '/www').clone("https://github.com/uuu/xxxxx.git")
```

-- I was doing it by adding dot (.) when cloning with terminal
git clone https://github.com/uuu/xxxxx.git .
/path/to

Thank you

**Comment by Byron** [comment, marked as answer]
I believe this invocation sets the working dir, but doesn't provide the actual `./path/to`.

Maybe try something along the lines of `git.Git(WEBDIR + domain.name + '/www').clone("https://github.com/uuu/xxxxx.git", "./path/to")`.

Lastly, you can see what GitPython actually executes by using the flags described [in the docs](https://gitpython.readthedocs.io/en/stable/tutorial.html#git-command-debugging-and-customization).

Even though I am closing this question, please feel free to keep adding follow ups as comments.
